cli/internal/cmd/configgenerate.go

```diff
@@ -31,12 +32,14 @@ func newConfigGenerateCmd() *cobra.Command {
 		RunE:              runConfigGenerate,
 	}
 	cmd.Flags().StringP("file", "f", constants.ConfigFilename, "path to output file, or '-' for stdout")
+	cmd.Flags().String("name", "constell", "name to use for cluster creation and configuration")
```

Comment:
Do we even want to have a flag? We're already generating a config for the user to fill out. We can also provide a default value in there. Also we already do the name validation when reading the config.